src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1313,17 +1313,12 @@ public override void VisitMemberInitializer(IMemberInitializerOperation operatio
             Visit(operation.Initializer, "Initializer");
         }
 
+        [Obsolete("ICollectionElementInitializerOperation has been replaced with IInvocationOperation and IDynamicInvocationOperation", error: true)]
```

Comment:
Use nameof

Reply:
NM. Saw your other comment. 

---
In reply to: [190641336](https://github.com/dotnet/roslyn/pull/26781#discussion_r190641336) [](ancestors = 190641336)